Makefile

```diff
@@ -25,8 +25,17 @@ else
 	data_home := ${XDG_DATA_HOME}
 endif
 
+STRIP := strip
+
+PREFIX := /usr/local
+
+install_dir = $(DESTDIR)/$(PREFIX)/share/steam/compatibilitytools.d/$(tool_dir)
+
 dev_install_dir = $(data_home)/Steam/compatibilitytools.d/$(tool_dir_dev)
 
+.PHONY: all
```

Comment:
`.PHONY` targets are already defined in first line.

Reply:
Huh? Wonder how I've missed it.

Reply:
Those phonies ;)